src/templates/core/proposals/assign/start_proposal.html

```diff
@@ -75,7 +80,15 @@
     </thead>
     <tbody>
     <tr>
-      <td>{{ proposal.requestor.username }} <sup> <a href="core/email.html" onclick="return popitup('/email/proposal/{{proposal.id}}/user/{{proposal.requestor.id}}/','proposal_{{proposal.requestor.id}}')"class="fa fa-envelope-o btn btn-xs btn-default"></a></sup></td>
+      <td>{{ proposal.requestor.username }}
+          <sup>
+              <a href="core/email.html"
+                 onclick="return popitup('/email/proposal/{{proposal.id}}/user/{{proposal.requestor.id}}/','proposal_{{proposal.requestor.id}}')"
+                 class="fa fa-envelope-o btn btn-xs btn-default">
+
+              </a>
```

Comment:
Good reformatting, but what the heck is this doing? Should the `href="core/email.html"` be there? Is there a reason we're not using the [`url`](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#url) tag? And why doesn't the link have some content (we could use the Bootstrap class `sr-only` or `text-hide` to hide it).